pkg/kubecost/allocation.go

```diff
@@ -87,6 +87,7 @@ type Allocation struct {
 	// asset on which the allocation was run. It is optionally computed
 	// and appended to an Allocation, and so by default is is nil.
 	ProportionalAssetResourceCosts ProportionalAssetResourceCosts `json:"proportionalAssetResourceCosts"` //@bingen:field[ignore]
+	SharedCostBreakdown            SharedCostBreakdowns           `json:"sharedCostBreakdown"`            //@bingen:field[ignore]
```

Comment:
@mbolt35 do bingen codecs have to be regenerated if we add an ignored field? My assumption is no, but I want to make sure.

Reply:
Nope, these are just transient fields though, so they won't be persisted. I'm guessing that was intended?

Reply:
Yep, that's intended.

Reply:
Thanks!